python/pyspark/sql/dataframe.py

```diff
@@ -3913,10 +3913,11 @@ def unpivot(
         ids : str, Column, tuple, list
             Column(s) to use as identifiers. Can be a single column or column name,
             or a list or tuple for multiple columns.
-        values : str, Column, tuple, list, optional
-            Column(s) to unpivot. Can be a single column or column name, or a list or tuple
-            for multiple columns. If specified, must not be empty. If not specified, uses all
-            columns that are not set as `ids`.
```

Comment:
Can you keep the indentation? It follows NumPy docstring.

Reply:
Should probably fix `melt` too

Reply:
hmm ok 
i will change the docs and remove the optional and the last line 
unless you want to change unpivot function to incorporate the functionality in the documentation